deepchem/models/layers.py

```diff
@@ -370,6 +370,380 @@ def call(self, inputs):
     return mol_features
 
 
+class GraphConvolutionLayer(tf.keras.layers.Layer):
```

Comment:
Would these layers be useful beyond `MolGan`? If not, perhaps better to rename `MolGanGraphConvolutionLayer`

Reply:
It is possible as it is based on Relational-GCN [(Schlichtkrull et al., 2017)](https://arxiv.org/abs/1703.06103).
I will live it up to you to decide. Let me know and I will change names for layers.

Reply:
Hmm, how about `RelationalGraphConvLayer` and we add a citation to the Schlichtkrull et al. paper in the references?

Reply:
I have checked code I could find and it is a bit different. I think using MolGAN is least confusing. I will change the code to MolGAN prefix.